pkg/ipam/crd.go

```diff
@@ -295,12 +295,8 @@ func (n *nodeStore) hasMinimumIPsInPool() (minimumReached bool, required, numAva
 	switch {
 	case n.ownNode.Spec.IPAM.MinAllocate != 0:
 		required = n.ownNode.Spec.IPAM.MinAllocate
-	case n.ownNode.Spec.ENI.MinAllocate != 0:
-		required = n.ownNode.Spec.ENI.MinAllocate
 	case n.ownNode.Spec.IPAM.PreAllocate != 0:
-		required = n.ownNode.Spec.IPAM.PreAllocate
-	case n.ownNode.Spec.ENI.PreAllocate != 0:
-		required = n.ownNode.Spec.ENI.PreAllocate
+		required = n.ownNode.Spec.IPAM.PreAllocate	
```

Comment:
This one still has a tab at the end. I probably missed it on my earlier comment, sorry.